src/components/NewTodoForm.tsx

```diff
@@ -12,9 +12,16 @@ const NewTodoForm: FC = () => {
     const [text, setText] = useState('');
     const [showModal, setShowModal] = useState(false);
     const dispatch = useAppDispatch();
+    const [error, setError] = useState("");
 
 
     const handleKeyPress = () => {
+        if (!text.match(/[@#~^`!-+()_|?{}[\]&*%<>\\$'"]/)) {
+            setError("");
+        } else {
```

Comment:
Why do you need this condition and setError(""), if it's false by default ?

Reply:
It's will error all the time, after one time setError("no special symbols allowed"), without setError("").